octavia-cli/README.md

```diff
@@ -348,8 +350,19 @@ $ octavia apply
 6. Run the unittest suite: `pytest --cov=octavia_cli`.
 7. Make sure the build passes (step 0) before opening a PR.
 
+## Telemetry
+This CLI has some telemetry tooling to send Airbyte some data about the usage of this tool.
+We use this data to measure the tool's adoption and detect common errors users encounter to improve it.
+The telemetry sends data about:
+* Which command was run (not the arguments or options used).
+* Success or failure of the command run and the error type.
+* The workspace id. It is unique to each Airbyte instance, but we can't match it to a username or email address.
+
+You can disable telemetry by setting the `OCTAVIA_ENABLE_TELEMETRY` environment to `false` or using the `--disable-telemetry` flag.
```

Comment:
Maybe the telemetry can also follow the boolean for tracking or not on the airbyte instance too?